content/docs/traffic-management/health-checks/backend.md

```diff
@@ -0,0 +1,111 @@
+---
+title: Backend health checks
+weight: 20
+---
+
+Automatically monitor the status of backends by configuring health checks.
+
+Health checks periodically and automatically assess the readiness of the Backend to receive requests. You can configure several settings, such as health thresholds and check intervals, that {{< reuse "/docs/snippets/kgateway.md" >}} uses to determine whether a service is marked as healthy or unhealthy. For more information, see the Envoy [health checking](https://www.envoyproxy.io/docs/envoy/latest/intro/arch_overview/upstream/health_checking#arch-overview-health-checking) documentation.
+
+## Before you begin
+
+{{< reuse "docs/snippets/prereq.md" >}}
+ 
+## Configure a health check for a Backend
+
+In the `healthCheck` section of a BackendConfigPolicy resource, specify settings for how you want the health check to perform for a Backend or Kubernetes service. The following example configures a simple set of HTTP health check settings for a service to get you started.
+
+```yaml
+apiVersion: gateway.kgateway.dev/v1alpha1
+kind: BackendConfigPolicy
+metadata:
+  name: healthcheck-policy
+  namespace: {{< reuse "/docs/snippets/namespace.md" >}}
+spec:
+  targetRefs:
+    - name: my-backend
+      group: ""
+      kind: Service
+  healthCheck:
+    healthyThreshold: 1
+    http:
+      path: /status/200
+    interval: 30s
+    timeout: 10s
+    unhealthyThreshold: 1
+```
+
+{{< reuse "/docs/snippets/review-table.md" >}} For more information about all of the settings that you can configure, see the [reference documentation](/docs/reference/api/health_check).
+
+| Setting | Description |
+| ------- | ----------- |
+| `grpc` | Configuration for a GRPC healthcheck. |
```

Comment:
I might be misunderstanding, but I don't see these grpc fields in the example? if they are omitted, maybe say so
```suggestion
| `grpc` | Optional configuration for a gRPC healthcheck. The example omits this field because the Backend is not a gRPC service. |
```